testing/it/ui-js/src/content/jcr_root/apps/core/wcm/tests/test-cases/FormOptions/v1/FormOptions.js

```diff
@@ -99,9 +99,9 @@ window.CQ.CoreComponentsIT.FormOptions.v1 = window.CQ.CoreComponentsIT.FormOptio
         // press the Add button
         .click("button :contains('Add')")
         // set the value
-        .fillInput("input[name='./value']", value)
+        .fillInput("input[name$='value']", value)
```

Comment:
I still find these selectors flimsy. If we came along with a new `<input type="text" value="./optionValue"/>` somewhere in the dialog, it would pass here.

Reply:
In the case of the FormOptions tests I went for the quick fix, to make the tests pass again. But I agree that we should define how in the future we are going to make the UI elements of dialogs 
selectable with either classes or data attributes.

Reply:
ok, could you make sure we have this effort tracked?

Reply:
created #297 as follow up issue